controllers/validators/schema/paths.js

```diff
@@ -17,4 +17,6 @@ module.exports = {
   messageId: { $ref : 'sch#/definitions/message/messageId' },
   story: { $ref: 'sch#/definitions/userTag/story' },
   relevance: { $ref: 'sch#/definitions/userTag/relevance' },
+  page: { $ref: 'sch#/definitions/query/page' },
+  page0: { $ref: 'sch#/definitions/query/page0' }, // page with offset = 0
```

Comment:
Could be changed to pageOffset0, pageNoOffset or other name, not very clear and doesn't look nice.